pkg/istio-agent/agent.go

```diff
@@ -306,8 +306,9 @@ func (a *Agent) initializeEnvoyAgent(ctx context.Context) error {
 		a.dynamicBootstrapWaitCh = make(chan error, 1)
 		// Simulate an xDS request for a bootstrap
 		// wait indefinitely and keep retrying with jittered exponential backoff
-		b := backoff.NewExponentialBackOff()
-		b.MaxElapsedTime = 30 * time.Second
+		b := backoff.NewExponentialBackOff(func(off *backoff.ExponentialBackOff) {
```

Comment:
Do we have use cases for this init funcs rather than just setting the fields directly. Looks like mostly used fields are initInterval and MaxElapsedTime . Any benefit of this init funcs?

Reply:
some others are for unit test